### ISSUE TITLE
Possibly change to pokeapi

### ISSUE THREAD

**Issue by ChaosTherum** [robinjoseph08/pokedextracker.com#209]
So I know that you put a lot of work into this but I figured I would mention in case you haven't heard of it. There is a database called pokeapi.co it is a pretty easy to use restful api if anything you could collect the data for your database from it. I was just thinking it might be easier to use than other sites. I'm a novice programmer and it was pretty straight forward plus they tend to stay pretty up to date. They have an absolute ton of data to use.

**Comment by robinjoseph08** [comment]
Hey @ChaosTherum, thanks for the suggestion! When we first started building this site out, we were planning on using pokeapi.co so that we can reduce the amount of custom work that we had to write. But as we started integrating, we realized early on that it probably was going to work out in the long run because of a few reasons:

- It would add a dependence onto a service that is maintained by an individual.
  - Any updates that we want to do on our site now need to wait for the API to update (e.g. gen VII hasn't been added yet so our support for it would be delayed considerably).
  - Since it's maintained by an individual, at any point if he decides to
